clang/lib/Driver/Driver.cpp

```diff
@@ -4953,6 +4952,11 @@ class OffloadingActionBuilder final {
                    Action::OFK_SYCL);
           }
           return SYCLDeviceOnly ? ABRT_Ignore_Host : ABRT_Success;
+
```

Comment:
NIT: Comment could be helpful here.

Reply:
This dead code.  I must have forgot to clean it out.